stl/inc/chrono

```diff
@@ -5171,6 +5176,180 @@ namespace chrono {
+                _THROW(format_error("Invalid type specification."));
+            }
+
+            _Chrono_specs<_CharT> _Conv_spec{._Modifier = _Char_mod, ._Type = _Char_type};
+            _Specs._Chrono_specs_list.push_back(_Conv_spec);
+        }
+
+        constexpr void _On_lit_char(_CharT _Lit_ch) {
+            _Chrono_specs<_CharT> _Lit_char_spec{._Lit_char = _Lit_ch};
+            _Specs._Chrono_specs_list.push_back(_Lit_char_spec);
+        }
+
+    protected:
+        _Chrono_format_specs<_CharT>& _Specs;
+    };
+
+    template <class _CharT, _Chrono_parse_spec_callbacks<_CharT> _Callbacks_type>
+    _NODISCARD constexpr const _CharT* _Parse_conversion_specs(
+        const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
+        ++_Begin; // move past %
+        if (_Begin == _End || *_Begin == '}') {
+            _THROW(format_error("Invalid format string."));
+        }
+
+        _CharT _Mod = '\0';
+        _CharT _Ch  = *_Begin;
+
+        if (_Ch == 'E' || _Ch == 'O') { // includes modifier
+            _Mod = _Ch;
+            ++_Begin;
+            if (_Begin == _End || *_Begin == '}') {
+                _THROW(format_error("Invalid format string - missing type after modifier."));
+            }
+        }
+
+        _CharT _Type = *_Begin;
+        _Callbacks._On_conversion_spec(_Mod, _Type);
+
+        return ++_Begin;
+    }
+
+    template <class _CharT, _Chrono_parse_spec_callbacks<_CharT> _Callbacks_type>
+    _NODISCARD constexpr const _CharT* _Parse_chrono_format_specs(
+        const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
+        if (_Begin == _End || *_Begin == '}') {
+            return _Begin;
+        }
+
+        _Begin = _Parse_align(_Begin, _End, _Callbacks);
+        if (_Begin == _End) {
+            return _Begin;
+        }
+
+        _Begin = _Parse_width(_Begin, _End, _Callbacks);
+        if (_Begin == _End) {
+            return _Begin;
+        }
+
+        if (*_Begin == '.') {
+            _Begin = _Parse_precision(_Begin, _End, _Callbacks);
+            if (_Begin == _End) {
+                return _Begin;
+            }
+        }
+
+        // chrono-spec
+        while (_Begin != _End && *_Begin != '}') {
+            if (*_Begin == '%') { // conversion-spec
+                if (_Begin + 1 == _End) {
+                    _THROW(format_error("Invalid format string- missing type after %"));
+                }
+
+                _CharT _Next_ch = *(_Begin + 1);
+                switch (_Next_ch) {
+                case 'n':
+                    _Callbacks._On_lit_char('\n');
+                    ++_Begin;
+                    break;
+                case 't':
+                    _Callbacks._On_lit_char('\t');
+                    ++_Begin;
+                    break;
+                case '%':
+                    _Callbacks._On_lit_char('%');
+                    ++_Begin;
+                    break;

```

Comment:
We should have test coverage verifying that `"A%nB%tC%%D"` is parsed as `'A'`, `'\n'`, `'B'`, `'\t'`, `'C'`, `'%'`, `'D'` (or something similar, ensuring that we don't "lose our place" while parsing these special literals).